compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/DocumentationAnalyzer.java

```diff
@@ -318,6 +302,20 @@ private void validateReferences(DocumentableNode documentableNode, AnalyzerData
         }
     }
 
+    private void validateObjectOrRecord(BLangTypeDefinition typeDefinition, List<BLangSimpleVariable> fields,
```

Comment:
We can improve the method name to indicate what this method actually does. Right now, it doesn't indicate what's being validated.